server/task_specs/feature-apply.py

```diff
@@ -0,0 +1,147 @@
+
+import csv
+import json
+
+import numpy as np
+from six import PY2
+
+SNIFFER_SIZE = 1024
```

Comment:
:+1:

I'd urge us to create an issue for this so we don't forget, and I'd say for such a small change with potential to save us future headaches, we should do it sooner rather than later.